test/python/Dockerfile

```diff
@@ -72,6 +72,17 @@ RUN install-tool poetry 1.8.3
 
 USER 1000
 
+#--------------------------------------
+# build: rye
+#--------------------------------------
+FROM build as rye
+
+RUN prepare-tool rye
```

Comment:
add to line 48

Reply:
What do you mean?

Reply:
use a single prepare tool call with multiple tool names